remote-execution/Part 03 - Remote Execution Exercise - Solution.ipynb

```diff
@@ -0,0 +1,213 @@
+{
```

Comment:
<p>Training this is pretty slow. Can we find an optimal numbers of epochs? Maybe 2 to 5 epochs? Also wonder if we should add an evaluation step here and encourage them to do that in the exercise.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/dropoutlabs/tf-world-tutorial/pull/7/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='8'/>

Reply:
I think that's ok, I won't run it for 15 epoch, just one or maybe we will just show them the result. It's the solution, if they want, they can go back to it later.